recipe/meta.yaml

```diff
@@ -1,35 +1,48 @@
-{% set version = "1.10" %}
-{% set sha256 = "bcbe8ada8c435a253b1fa438f505dd311832a88f52316fd7d30342b106a4242c" %}
-{% set runc_version = "0.1.0" %}
-{% set runc_sha256 = "78ae42905c0f26fcfa9353e48e1a001b5b5c3c94b33e2ceacc6bc84c23c61a88" %}
+{% set goname = "github.com/tianon/gosu" %}
+{% set version = "1.11" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+# The runc version much match the upstream Dockerfile
+# https://github.com/tianon/gosu/blob/master/Dockerfile
+{% set runc_goname = "github.com/opencontainers/runc" %}
+{% set runc_version = "1.0.0-rc5" %}
+{% set runc_pkg_src = ('src/'+runc_goname).replace("/",os.sep) %}
 
 package:
-  name: gosu
+  name: {{ name }}
```

Comment:
My suggestion to have the template recipe have the name set to go-{{ name }}, following the r-{{ name }} template. However, this is a legacy package and we are better off not changing things.

Another option, is to create a metapackage with he canonical go/conda package.